lib/puppet/provider/mongodb_replset/mongo.rb

```diff
@@ -275,6 +275,7 @@ def self.mongo_command(command, host=nil, retries=4)
 
     #Hack to avoid non-json empty sets
     output = "{}" if output == "null\n"
+    output = "{}" if output == "\nnull\n"
```

Comment:
What is the purpose of this change? Could you describe it as part of the commit message?